packages/browser/src/__tests__/surveys.test.ts

```diff
@@ -1537,4 +1537,33 @@ describe('surveys', () => {
             expect(() => getNextSurveyStep(survey, 0, 'some_string')).toThrow('The response type must be an integer')
         })
     })
+
+    describe('is enabled', () => {
+        it('starts effectively disabled', () => {
+            expect(surveys['_isSurveysEnabled']).toBe(undefined)
+        })
+
+        it('is enabled by having results in onRemoteConfig', () => {
+            expect(surveys['_isSurveysEnabled']).toBe(undefined)
+            surveys.onRemoteConfig({
+                surveys: true,
```

Comment:
this should be an array with surveys in it
https://github.com/PostHog/posthog-js/blob/00601e3725b249a0f9a9d7d69c583928e1bbfd5d/packages/browser/src/posthog-surveys.ts#L44